Exec/eb_run2d/inputs.2d.flow_past_cylinder-x-ebgodunov

```diff
@@ -16,6 +16,7 @@ ns.visc_tol=1.0e-11
 ns.use_godunov=1 
 ns.do_mom_diff=1
 ns.do_cons_trac=1
+ns.do_cons_trac2=1
```

Comment:
You have to have ns.do_trac2 =1 for do_cons_trac2 to be relevant. I'll fix the logic in NSB.

Reply:
I was finding that weird indeed. It also pops up in PeleLM where we never actually see the IAMR tracers.